corehq/motech/repeaters/models.py

```diff
@@ -347,7 +347,6 @@ def _migration_get_couch_model_class(cls):
     def _migration_get_fields(cls):
         return [
             "domain",
-            "format",
```

Comment:
This will probably be never required because we would always override it for all the types of Repeaters we want to update. We can also keep it to an empty array. I kept it to domain earlier to make some tests happy but later I have updated the tests to use the specific type of Repeaters rather than the generic Repeater class.

Reply:
:+1: probably makes sense to remove `"domain"` then.